src/Header/AcceptHeader.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * Spiral Framework.
+ *
+ * @license   MIT
+ * @author    Pavel Z
+ */
+
+declare(strict_types=1);
+
+namespace Spiral\Http\Header;
+
+/**
+ * Represents "Accept" header item.
+ */
+class AcceptHeader
+{
+    /** @var AcceptHeaderItem[] */
+    private $items = [];
+
+    /**
+     * AcceptHeader constructor.
+     * @param AcceptHeaderItem[] $items
+     */
+    public function __construct(array $items)
+    {
+        usort($items, [$this, 'sortItems']);
+
+        $this->items = $items;
+    }
+
+    /**
+     * @param string $string
+     * @return AcceptHeader
+     */
+    public static function fromString(string $string): self
+    {
+        $items = [];
+        $parts = explode(',', $string);
+        foreach ($parts as $part) {
+            try {
+                $items[] = AcceptHeaderItem::fromString(trim($part));
+            } catch (\Throwable $e) {
+            }
+        }
+
+        return new static($items);
+    }
+
+    /**
+     * @param string $mime
+     * @return bool
+     */
+    public function has(string $mime): bool
+    {
+        foreach ($this->items as $item) {
+            if ($item->getMime() === trim($mime)) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * @return AcceptHeaderItem[]
+     */
+    public function items(): array
+    {
+        return $this->items;
+    }
+
+    /**
+     * @param AcceptHeaderItem $a
+     * @param AcceptHeaderItem $b
+     * @return int
+     */
+    private function sortItems(AcceptHeaderItem $a, AcceptHeaderItem $b): int
```

Comment:
My suggestion is to remove temp variables `$qualityA`, `$qualityB`, `$countParamsA`, `$countParamsB` here because these are just getter-methods